salt/runners/vault.py

```diff
@@ -117,7 +118,57 @@ def _get_policies(minion_id, config):
 
     policies = []
     for pattern in policy_patterns:
-        policies.append(pattern.format(**mappings))
+        for expanded_pattern in _expand_pattern_lists(pattern, **mappings):
+            policies.append(expanded_pattern.format(**mappings))
 
     log.debug('{0} policies: {1}'.format(minion_id, policies))
     return policies
+
```

Comment:
We need one more newline here to match with the other separations between functions in this module.